### PR TITLE
Table view of People with Edit Functionality

### PR DESCRIPTION
**Pull request by Harshithj88**

### CODE REVIEW

**Review by soitgoes — COMMENTED** [review]
Nice work.  Let's add delete and add!

**Comment by Harshithj88** [comment]
Added Delete and Add Functionalities.

File: index.html [hunk 1]
```diff
@@ -1,23 +1,95 @@
...
+                <th>Name</th>
+                <th>Age</th>
+                <th>Is Minor</th>
+                <th>Action</th>
+            </tr>
+            {{#each people}}
+            <tr>
+                <td>{{name}}</td>
+                <td>{{age}}</td>
+                <td>{{isMinor}}</td>
```
Comment (soitgoes) [inline index.html:34]: Currently if you were to change the age it wouldn't update the isMinor.  You could change this to {{age < 21}} instead or you could handle the change in the edit.

**Review by soitgoes — COMMENTED** [review]
Looks good.  Nice work.  One comment.

**Pull request closed without merging**